ts2mls/js/src/main/scala/ts2mls/TSCompilerInfo.scala

```diff
@@ -7,7 +7,13 @@ import js.JSConverters._
 import ts2mls.types._
 
 object TypeScript {
-  private val ts = g.require("typescript")
+  private val ts: js.Dynamic = try g.require("typescript") catch {
+    case _ : Throwable => {
+      System.err.println("Cannot find typescript in the current directory. Please install by running \"npm install\".")
+      val process = g.require("process")
+      process.exit(-1)
+    }
+  }
```

Comment:
Is this change related to this PR? Or did I miss something?

Reply:
Not strongly related. But as required I added some error information. Maybe I should change the title of this PR, or simply use another PR to submit this change.

Reply:
I think it's fine.